easybuild/tools/options.py

```diff
@@ -2011,14 +2012,19 @@ def opts_dict_to_eb_opts(args_dict):
             prefix = '--'
         option = prefix + str(arg)
         value = args_dict[arg]
-        if isinstance(value, (list, tuple)):
-            value = ','.join(str(x) for x in value)
 
-        if value in [True, None]:
+        if str(arg) in allow_multiple_calls:
+            if not isinstance(value, (list, tuple)):
+                value = [value]
+            for v in value:
+                args.append(option + '=' + str(v))
+        elif value in [True, None]:
             args.append(option)
         elif value is False:
             args.append('--disable-' + option[2:])
         elif value is not None:
+            if isinstance(value, (list, tuple)):
+                value = ','.join(str(x) for x in value)
```

Comment:
Can we also cover this with an enhanced test?

Reply:
I think that is already covered with https://github.com/easybuilders/easybuild-framework/blob/3d29b0a55c803c6ac68786c77b8e65a9c90b814b/test/framework/options.py#L7239

Reply:
But i can add a separate test if needed